alt-tab-macos/ui/PreferencesPanel.swift

```diff
@@ -1,110 +1,211 @@
 import Cocoa
 
-class PreferencesPanel: NSPanel, NSTextViewDelegate {
-    var maxScreenUsage: NSTextView?
-    var windowPadding: NSTextView?
-    var cellPadding: NSTextView?
-    var cellBorderWidth: NSTextView?
-    var maxThumbnailsPerRow: NSTextView?
-    var iconSize: NSTextView?
-    var fontHeight: NSTextView?
-    var interItemPadding: NSTextView?
-    var tabKeyCode: NSTextView?
-    var windowDisplayDelay: NSTextView?
+class PreferencesPanel: NSPanel {
+    // ui: base layout
+    let panelWidth = CGFloat(400)
+    let panelHeight = CGFloat(400) // gets auto adjusted to content height
+    let panelPadding = CGFloat(40)
+    let panelWidthToLabelRatio = CGFloat(0.5)
```

Comment:
Not the clearest concept 😅Does this mean 50% of the width of the panel is reserved for labels?

Reply:
Yes, currently its that much. This is used for the derived value to make the labels reserve the same amount of horizontal space. 

At the moment the label wording has to be long to get the message across. We can shorten them once we get tooltips done to explain settings in more details. This will allow us to reserve less space for the labels.